### PR TITLE
XmlTextAttribute test

### PR DESCRIPTION
**Pull request by imcarolwang**
Hi @huanwu , @shmao , case in this PR is part of old portable library test project, it contains more tests than ported here but I found most of them already covered in the existing Core test project, if you feel this test is also covered already, ignore the merge. Thanks.

### CODE REVIEW

File: src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs [hunk 1]
```diff
@@ -4182,6 +4182,26 @@ public static void Xml_Soap_TypeWithReadOnlyMyCollectionProperty()
...
+        myGroup2.TypeOfGroup = GroupType.Medium;
+        var actual2 = SerializeAndDeserialize(myGroup2, @"<?xml version=""1.0""?><Group2WithXmlTextAttr xmlns:xsi=""http://www.w3.org/2001/XMLSchema-instance"" xmlns:xsd=""http://www.w3.org/2001/XMLSchema"">Medium</Group2WithXmlTextAttr>");
+        Assert.Equal(myGroup2.TypeOfGroup, actual2.TypeOfGroup);
+
+        var myGroup3 = new Group3WithXmlTextAttr();
+        var actual3 = SerializeAndDeserialize(myGroup3, @"<?xml version=""1.0""?><Group3WithXmlTextAttr xmlns:xsi=""http://www.w3.org/2001/XMLSchema-instance"" xmlns:xsd=""http://www.w3.org/2001/XMLSchema"">2017-04-20T03:08:15Z</Group3WithXmlTextAttr>");
+        Assert.Equal(myGroup3.CreationTime, actual3.CreationTime);
+
+        var myGroup4 = new Group4WithXmlTextAttr();
+        Assert.Throws(typeof(System.InvalidOperationException), () => { SerializeAndDeserialize(myGroup4, null, null, true); });
```
Comment (ViktorHofer) [inline src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs]: nit: Use generic Throws<Type> method

File: src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs [hunk 2]
```diff
@@ -4182,6 +4182,26 @@ public static void Xml_Soap_TypeWithReadOnlyMyCollectionProperty()
         Assert.True(value.Collection.SequenceEqual(actual.Collection));
     }
 
+    [Fact]
+    public static void Xml_XmlTextAttributeTest()
```
Comment (safern) [inline src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs]: Does this test pass in Desktop as well or it also tests .Net Core specific behavior? 

You could verify this by doing the following:
`build.cmd -framework:netfx`
`msbuild <path-to-test-csproj> /t:rebuildandtest /p:targetgroup=netfx`

File: src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs (hunk 1 above)
Comment (imcarolwang) [inline src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs]: Thank you. Will update.

File: src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs (hunk 2 above)
Comment (imcarolwang) [inline src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs]: Verified, it passed on Desktop as well.

Comment (safern) [inline src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs]: Thanks for verifying!

**Comment by shmao** [comment]
@imcarolwang thanks for the PR!

**Pull request merged**